boards: shields: nxp_m2_wifi_bt: doc: Update doc (#92429)
zephyrproject-rtos/zephyr: boards/shields/nxp_m2_wifi_bt/doc/index.rst

Integration Platform
********************

> This shield is validated and tested for use with the host platform listed below.
> While the shield can be used with other host platforms, NXP may not actively test
> or validate these combinations.

Reviewer: ```suggestion
This shield is validated and tested for use with the host platform listed below.
While it may be used with other host platforms, other combinations are not actively tested or validated.
```

Author: Updated
